.releaserc

```diff
@@ -21,20 +21,20 @@
       { name: "develop", prerelease: "beta", channel: "beta" },
     ],
   plugins:
+  [
+    "@semantic-release/commit-analyzer",
     [
-      "@semantic-release/commit-analyzer",
-      [
-        "@semantic-release/exec",
-        {
-          "verifyReleaseCmd": "echo \"version=${nextRelease.version}\" >> $GITHUB_OUTPUT",
-          "successCmd": "echo \"new_release_published=${'true'}\" >> $GITHUB_OUTPUT",
-        }
-      ],
-      "@semantic-release/release-notes-generator",
-      ["@semantic-release/git", {
-            "assets": ["action.yml"],
-            "message": "chore(release): ${nextRelease.version}\n\n${nextRelease.notes}"
-        }],
-      "@semantic-release/github",
+      "@semantic-release/exec",
+      {
+        "verifyReleaseCmd": "echo \"version=${nextRelease.version}\" >> $GITHUB_OUTPUT",
+        "successCmd": "echo \"new_release_published=${'true'}\" >> $GITHUB_OUTPUT",
+      }
     ],
+    "@semantic-release/release-notes-generator",
+    ["@semantic-release/git", {
+          "assets": ["action.yml"],
+          "message": "chore(release): ${nextRelease.version}\n\n${nextRelease.notes}"
```

Comment:
Can you please add "[ci skip]" in the end of the message? This way it will not trigger another CI run from the bot commit.

Reply:
In the add-ons bit does bot commit anything.